### PR TITLE
Implement REST API with OpenAPI and Connexion

### PR DESCRIPTION
**Pull request by eugene-babichenko**
**TBD**:

* Admin API (validators and on-chain settings)
* Enable/disable methods in API

### CODE REVIEW

File: remme/rest_api/personal.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import os
+import re
+from connexion import NoContent
+from sawtooth_signing import create_context
+
+
+from remme.settings import KEY_DIR
+
+
+def search():
```
Comment (bogomazov) [inline remme/rest_api/personal.py]: где это используется?

File: remme/rest_api/certificate.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            mgf=padding.MGF1(hashes.SHA256()),
+            salt_length=padding.PSS.MAX_LENGTH
+        ),
+        hashes.SHA256()
+    )
+
+    status, _ = client.store_certificate(crt_bin, rem_sig, crt_sig.hex())
+
+    return {'certificate': crt_export.decode('utf-8'),
+            'privkey': key_export.decode('utf-8'),
```
Comment (bogomazov) [inline remme/rest_api/certificate.py]: 'priv_key' мжт? мелочь но приятно

File: remme/rest_api/openapi.yml [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+  /token:
+    post:
+      summary: Send tokens from the current user to another user
+      parameters:
+        - name: payload
+          in: body
+          required: true
+          schema:
+            properties:
+              pubkey_to:
```
Comment (bogomazov) [inline remme/rest_api/openapi.yml]: тоже `pub_key_to` ?

File: remme/rest_api/personal.py (hunk 1 above)
Comment (bogomazov) [reply remme/rest_api/personal.py]: @eugene-babichenko чекнишь этот?

Comment (eugene-babichenko) [reply remme/rest_api/personal.py]: `search()` is the name of the function to which Connexion mapper automatically binds `GET` requests with no arguments. Quite non-obvious. The purpose of this is to get the whole list of key pairs.

**Pull request merged**